.travis.yml

```diff
@@ -3,16 +3,16 @@ services:
   - docker
 language: python
 cache: pip
-python:
-  - "2.7"
-  - "3.6"
 
 env:
-  - ORPC_TEST_VERSION="8.0" DOCKER_PG_IMAGE="postgres:9.4"
-  - ORPC_TEST_VERSION="9.0" DOCKER_PG_IMAGE="postgres:9.4"
-  - ORPC_TEST_VERSION="10.0" DOCKER_PG_IMAGE="postgres:9.4"
-  - ORPC_TEST_VERSION="11.0" DOCKER_PG_IMAGE="postgres:9.4"
-  - ORPC_TEST_VERSION="12.0" DOCKER_PG_IMAGE="postgres:10"
+  - ORPC_TEST_VERSION="8.0" DOCKER_PG_IMAGE="postgres:9.4" TRAVIS_PYTHON_VERSION="2.7"
+  - ORPC_TEST_VERSION="9.0" DOCKER_PG_IMAGE="postgres:9.4" TRAVIS_PYTHON_VERSION="2.7"
+  - ORPC_TEST_VERSION="10.0" DOCKER_PG_IMAGE="postgres:9.4" TRAVIS_PYTHON_VERSION="2.7"
+  - ORPC_TEST_VERSION="11.0" DOCKER_PG_IMAGE="postgres:9.4" TRAVIS_PYTHON_VERSION="2.7"
+  - ORPC_TEST_VERSION="11.0" DOCKER_PG_IMAGE="postgres:9.4" TRAVIS_PYTHON_VERSION="3.6"
+  - ORPC_TEST_VERSION="12.0" DOCKER_PG_IMAGE="postgres:10" TRAVIS_PYTHON_VERSION="3.6"
+  - ORPC_TEST_VERSION="13.0" DOCKER_PG_IMAGE="postgres:10" TRAVIS_PYTHON_VERSION="3.6"
+  - ORPC_TEST_VERSION="14.0" DOCKER_PG_IMAGE="postgres:10" TRAVIS_PYTHON_VERSION="3.6"
```

Comment:
I think we are not testing the same thing by doing that: we could have OdooRPC installed with Python 2.7, and performing requests against an Odoo instance running with Python 3.6.
But anyway, it's time to drop the compatibility with Python 2.7 so I guess it's OK :)